obspy/io/gse2/libgse2.py

```diff
@@ -465,13 +465,33 @@ def parse_sta2(line):
     """
     header = {}
     try:
+        lat = line[15:24].strip()
+        if lat:
+            lat = float(lat)
+        else:
+            lat = None
+        lon = line[25:35].strip()
+        if lon:
+            lon = float(lon)
+        else:
+            lon = None
+        lon = lon and float(lon)
```

Comment:
Thanks. Left-over from shoving things around. Initially wanted to avoid the ugly and longish `if/else`s. But as it turns out..

```
lon = "0"
lon and float(lon) or None
```

.. evaluates to `None`, which surprised me a bit.